readme.md

```diff
@@ -23,12 +23,29 @@ See the [source](index.js) for supported symbols.
 ```js
 const figures = require('figures');
 
+console.log(figures('✔︎ check'));
+// On real OSes:  ✔︎ check
+// On Windows:    √ check
+
 console.log(figures.tick);
 // On real OSes:  ✔︎
 // On Windows:    √
 ```
 
 
+## API
+
+### figures(input)
+
+Returns the `input` with replaced fallback unicode symbols.
```

Comment:
Not sure about this one. It seems like it's always replacing to fallback unicode symbols but it depends on the OS really. So one of the following might be better.

> Returns the `input` with replaced fallback unicode symbols on Windows.

> Returns the `input` with OS supported unicode symbols.

Or other suggestions are more then welcome as well.